src/OrleansRuntime/Placement/IPlacementContext.cs

```diff
@@ -26,6 +26,8 @@ internal interface IPlacementContext
 
         SiloAddress LocalSilo { get; }
 
+        SiloStatus LocalSiloStatus { get; }
```

Comment:
we recently took our warning count to 0, so be sue to add XML comments for public members